mne/io/ctf/markers.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+class Markers:
```

Comment:
Dont write a class for this. Just use a dumb plain private function that returns an annotation object. Thx